### ISSUE TITLE
Migrate vendor components to gymcms

### ISSUE THREAD

**Issue by rediris** [gymnasium/tracker#201]
A student from Malaysia reported being unable to properly view our site due to their ISP blocking maxcdn.

<img width="655" alt="Screen Shot 2021-11-29 at 9 42 41 AM" src="https://user-images.githubusercontent.com/1010395/143888372-e28a1016-ebd8-48f1-ae4a-4c701fe3cb6c.png">
